### PR TITLE
Allow vertexProgram and fragmentProgram to be nullptrs

### PR DESCRIPTION
**Pull request by EXtremeExploit**
self explanatory

### CODE REVIEW

File: vita3k/modules/SceGxm/SceGxm.cpp [hunk 1]
```diff
@@ -2101,9 +2100,8 @@ EXPORT(int, sceGxmSetVertexDefaultUniformBuffer, SceGxmContext *context, Ptr<con
 
 EXPORT(void, sceGxmSetVertexProgram, SceGxmContext *context, Ptr<const SceGxmVertexProgram> vertexProgram) {
     assert(context);
-    assert(vertexProgram);
 
-    if (!context || !vertexProgram)
+    if (!context)
```
Comment (Zangetsu38) [inline vita3k/modules/SceGxm/SceGxm.cpp:2104]: wrong

File: vita3k/modules/SceGxm/SceGxm.cpp [hunk 2]
```diff
@@ -1836,9 +1836,8 @@ EXPORT(int, sceGxmSetFragmentDefaultUniformBuffer, SceGxmContext *context, Ptr<c
 
 EXPORT(void, sceGxmSetFragmentProgram, SceGxmContext *context, Ptr<const SceGxmFragmentProgram> fragmentProgram) {
     assert(context);
-    assert(fragmentProgram);
 
-    if (!context || !fragmentProgram)
+    if (!context)
```
Comment (Zangetsu38) [inline vita3k/modules/SceGxm/SceGxm.cpp:1840]: wrong here too, and you can remove both assert, no needed

**Comment by Zangetsu38** [comment]
in my side, probleme for this game don't relate shader, but relate path device
image

**Comment by EXtremeExploit** [comment]
My bad, turns out i confused my changes, it makes the game boot in this PR https://github.com/Vita3K/Vita3K/pull/1156 and then the program pointers problem, i confused the order

**Comment by Zangetsu38** [comment]
> 
> 
> My bad, turns out i confused my changes, it makes the game boot in this PR #1156 and then the program pointers problem, i confused the order

oh i see thx

**Comment by Zangetsu38** [comment]
Thx for tyhis works, but no have any sense, game can go ingame without this pr, like i have explain, frag/vert null is sometime normal, code using only skip shader, return  if frag/vert null already skip it, for other, is just random crash on fp->hash null, but is relate other things, stabitily etc

**Pull request closed without merging**